Dockerfile

```diff
@@ -23,14 +23,15 @@ FROM alpine:3.17
 LABEL org.label-schema.description="Useful network related tools"
 LABEL org.label-schema.vendor=travelping.com
 LABEL org.label-schema.copyright=travelping.com
-LABEL org.label-schema.version=1.12.0
+LABEL org.label-schema.version=1.13.0
```

Comment:
keep that at 1.12.0 for now, there will be other changes coming to justify a bump.

alpine:3.20 was just released, which means bird:2.15 (alpine:3.20) instead of bird:2.10 (alpine:3.17)

Reply:
I used 3.20 as a base.

As for versioning - since we added a new feature (bird2) and a new alpine image as a base - why shouldn't it be 1.13? Do you have any version suggestion?

Reply:
ah, i see now.

then the commit / pr need to transport that information. with that, 1.13.0 is ok